README.md

```diff
@@ -11,15 +11,19 @@
 [![check_spelling](https://github.com/GSA/usagov-benefits-eligibility/actions/workflows/check_spelling.yml/badge.svg)](https://github.com/GSA/usagov-benefits-eligibility/actions/workflows/check_spelling.yml)
 
 
-BEARS (Benefits Eligibility Awareness Resources)
-will be a proactive notification service that provides
-timely and personalized information about benefits
-eligibility, contextualized by life events. Before we can
-do notifications and potentially proactive notifications,
-we need to be able to estimate eligibility and figure out a
-way to get people to sign up and store their information.
-We also need to establish relationships with benefits
-programs so we can keep eligibility information up to date.
+The BEARS (Benefits Eligibility Awareness Resources)
+is an interactive tool that enables the public to learn
+about benefits available to them depending on their life
+experiences. BEARS aims to simplify the experience of
+finding and understanding life experience based benefits,
+in both English and Spanish.
+
+Currently, the only supported life experience is death of
```

Comment:
we do have https://benefits-tool.usa.gov/disability/? and https://benefits-tool.usa.gov/retirement/?

should we update the language here to reflect?

Reply:
Thanks for this comment Scott! Changes have been made